packages/muelu/src/Misc/MueLu_SchurComplementFactory_def.hpp

```diff
@@ -181,8 +181,8 @@ SchurComplementFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::ComputeSchurC
     if (!A11.is_null()) {
       S = MatrixFactory::BuildCopy(A11);
     } else {
-      S = MatrixFactory::Build(A11->getRowMap(), 10 /*A11->getLocalMaxNumRowEntries()*/);
-      S->fillComplete(A11->getDomainMap(), A11->getRangeMap());
+      TEUCHOS_TEST_FOR_EXCEPTION(true, Exceptions::RuntimeError,
+                                 "MueLu::SchurComplementFactory::Build: Constructing Schur complement for matrix with non-existent diagonal block.");
```

Comment:
@cgcgcg Isn't the case of a zero block in the bottom right place of a 2x2 block matrix the perfect textbook example of a saddle point system suitable for Schur complement approaches?

Reply:
Not in this case. Further up, this check is false:
https://github.com/trilinos/Trilinos/blob/7252bec780885da9091347ce3ea88e7e14daae5b/packages/muelu/src/Misc/MueLu_SchurComplementFactory_def.hpp#L134
and just above this check is false
https://github.com/trilinos/Trilinos/blob/7252bec780885da9091347ce3ea88e7e14daae5b/packages/muelu/src/Misc/MueLu_SchurComplementFactory_def.hpp#L169